src/components/AutoSuggest/SuggestList.js

```diff
@@ -0,0 +1,96 @@
+import React, { Fragment, useEffect, useRef, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { themeColor, themeSpacing, Icon } from '@datapunt/asc-ui';
+import { ChevronRight } from '@datapunt/asc-assets';
+
+const StyledList = styled.ul`
+  border: 1px solid ${themeColor('tint', 'level5')};
+  border-top: 0;
+  padding: 0;
+  margin: 0;
+`;
+
+const Li = styled.li`
+  line-height: 20px;
```

Comment:
do we want to work with fixed values here? themeSpacing?

Reply:
Indeed, good catch. I'll change it.